distributed/tests/test_utils.py

```diff
@@ -625,6 +625,10 @@ def test_lru():
     assert len(l) == 3
     assert list(l.keys()) == ["c", "a", "d"]
 
+    # clear
+    l.clear()
+    assert len(l) == 0
+    assert list(l.keys()) == []
```

Comment:
I wouldn't mind dropping this part of the test and just relying on the integration test that you have in `test_client.py`.  I think that that will capture the intent as effectively and be robust to future internal changes.

Reply:
addressed w/ https://github.com/dask/distributed/pull/3993/commits/80895f627c9abf2517c0d337994ed9f466c02b40

Reply:
addressed w/ 80895f6